setup.py

```diff
@@ -34,6 +33,7 @@
 else:
     _install_requires = [
         'pylint>=1.0',
+        'pylint-plugin-utils>=0.2.4',
```

Comment:
Just made a new minor release of that too to include the LICENSE file there as well but it otherwise hasn't changed in years!